Error running storybook in dev mode locally

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Victory version

latest

### Code Sandbox link

_No response_

### Bug report

Running `pnpm storybook:dev` locally fails due to the following error:

```markdown
SB_CORE-SERVER_0002 (CriticalPresetLoadError): Storybook failed to load the following preset: @storybook/react-webpack5/preset.

Please check whether your setup is correct, the Storybook dependencies (and their peer dependencies) are installed correctly and there are no package version clashes.

If you believe this is a bug, please open an issue on Github.

Error: Cannot find module '@storybook/react-webpack5/preset'
Require stack:
- /Users/djm/Library/Caches/pnpm/dlx/x6lev33jaseh2jcerd6yyuj4ly/193887ba168-8f6c/node_modules/.pnpm/@storybook+core@8.4.6/node_modules/@storybook/core/dist/common/index.cjs
- /Users/djm/Library/Caches/pnpm/dlx/x6lev33jaseh2jcerd6yyuj4ly/193887ba168-8f6c/node_modules/.pnpm/storybook@8.4.6/node_modules/storybook/dist/proxy.cjs
- /Users/djm/Library/Caches/pnpm/dlx/x6lev33jaseh2jcerd6yyuj4ly/193887ba168-8f6c/node_modules/.pnpm/storybook@8.4.6/node_modules/storybook/bin/index.cjs
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._resolveFilename (/Users/djm/Library/Caches/pnpm/dlx/x6lev33jaseh2jcerd6yyuj4ly/193887ba168-8f6c/node_modules/.pnpm/esbuild-register@3.6.0_esbuild@0.24.0/node_modules/esbuild-register/dist/node.js:4794:36)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at interopRequireDefault (/Users/djm/Library/Caches/pnpm/dlx/x6lev33jaseh2jcerd6yyuj4ly/193887ba168-8f6c/node_modules/.pnpm/@storybook+core@8.4.6/node_modules/@storybook/core/dist/common/index.cjs:11666:11)
    at getContent (/Users/djm/Library/Caches/pnpm/dlx/x6lev33jaseh2jcerd6yyuj4ly/193887ba168-8f6c/node_modules/.pnpm/@storybook+core@8.4.6/node_modules/@storybook/core/dist/common/index.cjs:16436:10)
    at loadPreset (/Users/djm/Library/Caches/pnpm/dlx/x6lev33jaseh2jcerd6yyuj4ly/193887ba168-8f6c/node_modules/.pnpm/@storybook+core@8.4.6/node_modules/@storybook/core/dist/common/index.cjs:16442:51)
    at /Users/djm/Library/Caches/pnpm/dlx/x6lev33jaseh2jcerd6yyuj4ly/193887ba168-8f6c/node_modules/.pnpm/@storybook+core@8.4.6/node_modules/@storybook/core/dist/common/index.cjs:16488:24
    at Array.map (<anonymous>)

More info:

    at loadPreset (/Users/djm/Library/Caches/pnpm/dlx/x6lev33jaseh2jcerd6yyuj4ly/193887ba168-8f6c/node_modules/.pnpm/@storybook+core@8.4.6/node_modules/@storybook/core/dist/common/index.cjs:16477:13)

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.
```


### Steps to reproduce

```markdown
1. Clone repo locally
2. pnpm install
3. pnpm storybook:dev
```


### Expected behavior

```markdown
Storybook should open in the browser
```


### Actual behavior

```markdown
Storybook fails to load due to build error
```


### Environment

```markdown
- Device: Mac
- OS: 15.1.1
- Node: v20.11.1
- pnpm: 9.13.0
```

--- a/.storybook/main.ts
+++ b/.storybook/main.ts
@@ -5,9 +5,12 @@ const path = require("path");
 const ROOT = path.resolve(__dirname, "..");
 const STORIES = path.resolve(ROOT, "stories");
 
+const getAbsolutePath = (packageName: string): any =>
+  path.dirname(require.resolve(path.join(packageName, 'package.json')));
+
 const config: StorybookConfig = {
   addons: [
-    "@storybook/addon-essentials",
+    getAbsolutePath("@storybook/addon-essentials"),
     {
       name: "@storybook/addon-storysource",
       options: {
@@ -20,12 +23,12 @@ const config: StorybookConfig = {
         },
       },
     },
-    "@storybook/addon-webpack5-compiler-swc",
-    "@chromatic-com/storybook",
+    getAbsolutePath("@storybook/addon-webpack5-compiler-swc"),
+    getAbsolutePath("@chromatic-com/storybook"),
   ],
 
   framework: {
-    name: "@storybook/react-webpack5",
+    name: getAbsolutePath("@storybook/react-webpack5"),
     options: {
       builder: {},
     },

